docs/source/tutorials/keras-image-classifiers.rst

```diff
@@ -0,0 +1,12 @@
+.. _keras-image-classifiers-tutorial:
+
+.. note::
+
+    This tutorial is intended to be run in an IPython notebook.
+    It is also available as a notebook file here_.
+
+.. _here: https://github.com/TeamHG-Memex/eli5/blob/master/notebooks/keras-image-classifiers.ipynb
+
+.. 
+    note that we need to rewrite the image paths when the notebook is converted to rst
```

Comment:
Can this be automated? We already have some similar fixes in the script that converts notebooks.
Also this note probably shouldn't be in user-visible docs?

Reply:
> Can this be automated? We already have some similar fixes in the script that converts notebooks.

Thanks for bringing this up. Completely missed that there is a `update-notebooks.sh` script. Will add commands to that.



>  Also this note probably shouldn't be in user-visible docs?

This is a Sphinx "comment" so it wouldn't be visible to the user. But since we have a script there is no need for it.

Reply:
> This is a Sphinx "comment" so it wouldn't be visible to the user. But since we have a script there is no need for it.
 TIL about Sphinx comments, thanks!